### PR TITLE
Update ext/simplexml methodsynopses based on stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/simplexml/functions/simplexml-import-dom.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
   &reftitle.description;
   <methodsynopsis>
    <type class="union"><type>SimpleXMLElement</type><type>null</type></type><methodname>simplexml_import_dom</methodname>
-   <methodparam><type>DOMNode</type><parameter>node</parameter></methodparam>
-   <methodparam choice="opt"><type>string</type><parameter>class_name</parameter><initializer>"SimpleXMLElement"</initializer></methodparam>
+   <methodparam><type class="union"><type>SimpleXMLElement</type><type>DOMNode</type></type><parameter>node</parameter></methodparam>
+   <methodparam choice="opt"><type class="union"><type>string</type><type>null</type></type><parameter>class_name</parameter><initializer>SimpleXMLElement::class</initializer></methodparam>
```
Comment (kocsismate) [inline reference/simplexml/functions/simplexml-import-dom.xml:13]: the param type didn't change here, only the documentation was incomplete

File: reference/simplexml/functions/simplexml-load-file.xml [hunk 2]
```diff
@@ -12,9 +12,9 @@
   <methodsynopsis>
    <type class="union"><type>SimpleXMLElement</type><type>false</type></type><methodname>simplexml_load_file</methodname>
    <methodparam><type>string</type><parameter>filename</parameter></methodparam>
-   <methodparam choice="opt"><type>string</type><parameter>class_name</parameter><initializer>"SimpleXMLElement"</initializer></methodparam>
+   <methodparam choice="opt"><type class="union"><type>string</type><type>null</type></type><parameter>class_name</parameter><initializer>SimpleXMLElement::class</initializer></methodparam>
```
Comment (kocsismate) [inline reference/simplexml/functions/simplexml-load-file.xml:15]: the same here

**Comment by cmb69** [comment]
Thank you!

**Comment by cmb69** [comment]
Applied as 5916fd8f742127259912e6510904451849928020.

**Pull request closed without merging**